src/App.css

```diff
@@ -55,3 +54,11 @@ tr {
 .card {
   width: 18rem;
 }
+
+.restaurantButton {
+  margin: 10px;
+}
+
+.highlight-row {
+  background-color: yellow;
+}
```

Comment:
Good use of CSS to add a highlight to a row, and good use of clear naming to make it obvious what this class refers to.